src/v/cloud_storage/tests/segment_meta_cstore_test.cc

```diff
@@ -808,6 +808,7 @@ BOOST_AUTO_TEST_CASE(test_segment_meta_cstore_full_contains) {
 void test_cstore_prefix_truncate(size_t test_size, size_t max_truncate_ix) {
     // failing seed:
     // std::istringstream{"10263162"} >> random_generators::internal::gen;
+    BOOST_REQUIRE_GE(test_size, 2);
```

Comment:
Just a thought: do we have any tests that attempt to truncate the entire cstore?

Reply:
good idea, just added